src/server/controllers/users.controller.js

```diff
@@ -0,0 +1,38 @@
+'use struct'
+var User = require('mongoose').model('User')
```

Comment:
I was told that variable name should be camel case and that upper case at first letter should use only when it is a class name.